core/controller-decorator/src/decorator/mcp/MCPController.ts

```
@@ -0,0 +1,33 @@
+import { ControllerType, MCPControllerParams, AccessLevel, EggProtoImplClass } from '@eggjs/tegg-types';
+import { PrototypeUtil, SingletonProto } from '@eggjs/core-decorator';
+import ControllerInfoUtil from '../../util/ControllerInfoUtil';
+import { StackUtil } from '@eggjs/tegg-common-util';
+import { MCPInfoUtil } from '../../../src/util/MCPInfoUtil';
+
+export function MCPController(param?: MCPControllerParams) {
+  return function(constructor: EggProtoImplClass) {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)